### PR TITLE
feat: Replace product card carousel with a single static image

### LINKED ISSUE #55: Replace Carousel with Single Image in Product Cards
## 📘 Description

The current product cards use an image carousel, which can impact page loading times when there are many products and images. To improve performance and simplify the design, replace the carousel with a single static image for each card. Ensure the design remains visually appealing and responsive.

## 🔢 Steps

1. Replace the image carousel in the product card component with a single static image.
2. Verify that the updated design aligns well with the existing layout.
3. Ensure the new product card design is responsive across different screen sizes.

## ✅ Acceptance Criteria

- The new product card design should use a single static image for each card.
- The design should remain visually appealing and align with the existing layout.
- The product card should be responsive and adapt correctly to different screen sizes.

## 🌎 References

Not applicable.

## 📜 Additional Notes

- Consider choosing the first image of each product as the static image.

⚠ **Do not apply until the ODHack begins.**
⚠ **Read our guidelines before applying.**

### PR DESCRIPTION
**Pull request by Belloabraham121**
# 📝 feat: Replace product card carousel with a single static image

## 🛠️ Issue
- Closes #55 

## 📖 Description
- The current product cards use an image carousel, which can impact page loading times when there are many products and images. To improve performance and simplify the design, replace the carousel with a single static image for each card. Ensure the design remains visually appealing and responsive.

## ✅ Changes made
- Removed the image carousel from product card components.
- Displayed the first product image as a static image.
- Updated the design to ensure it aligns with the existing layout.
- Verified responsiveness and ensured proper behavior across different screen sizes.

## 🖼️ Media (screenshots/videos)
- Removed the carousel
image

- Product card is still responsive 
image

## 📜 Additional Notes
- While testing responsiveness, I observed that the Connect Wallet button is not responsive. This issue might need to be addressed separately.

### CODE REVIEW

File: frontend/app/components/ui/carousel.tsx
Comment (derianrddev) [inline frontend/app/components/ui/carousel.tsx:1]: You can run the command `npx shadcn@latest add carousel` to restore the carousel component to its original state. This component will be needed later in the project, so it should not be modified.

File: frontend/app/components/ui/image-carrousel.tsx
Comment (derianrddev) [inline frontend/app/components/ui/image-carrousel.tsx:1]: Leave this file as it was originally, for the same reason.

**Review by derianrddev — CHANGES_REQUESTED** [review]

**Comment by derianrddev** [comment]
@Belloabraham121 On the Marketplace page (`app/marketplace/page.tsx`), remove the `ImageCarousel` from the product cards and instead add an image for each product. That’s what we wanted.

**Comment by danielcdz** [comment]
@Belloabraham121 let us know if you need any help or clarification

**Comment by Belloabraham121** [comment]
@derianrddev Kindly review

**Review by derianrddev — APPROVED** [review]

**Review by danielcdz — APPROVED** [review]
LGTM

**Pull request merged**